tests/snmp/test_snmp_phy_entity.py

```diff
@@ -344,6 +344,8 @@ def test_psu_info(duthosts, enum_rand_one_per_hwsku_hostname, snmp_physical_enti
     """
     snmp_physical_entity_info = snmp_physical_entity_and_sensor_info["entity_mib"]
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    if not duthost.is_supervisor_node():
+        pytest.skip("Not supported on non supervisor node")
```

Comment:
This test is getting skipped based on the check below which looks for PSU_INFO in state db.
Do we need another skip here?
The current skip will look like this: "Fan information does not exist in DB, skipping this test"

Reply:
Hi @SuvarnaMeenakshi , this PR is undoing an earlier change which causes all these tests to skip. we can add more checks later on top of thise